atomicfu/src/commonMain/kotlin/kotlinx/atomicfu/AtomicFU.common.kt

```diff
@@ -23,6 +23,9 @@ import kotlin.reflect.KProperty0
  */
 public expect fun <T> atomic(initial: T, trace: TraceBase = None): AtomicRef<T>
 
+// Binary compatibility
```

Comment:
Please drop a comment "Binary compatibility with IR, should be removed with Kotlin 1.5 release"